docs/Installing.asciidoc

```diff
@@ -34,12 +34,24 @@ zypper in openQA-bootstrap
 /usr/share/openqa/script/openqa-bootstrap
 -------------------------------------------------------------------------------
 
+If you happen to be using an old Leap 15.0 system which doesn't yet have the
```

Comment:
I would just write 'If you still use an old Leap 15.0 system which doe**s no**t **already** have the ...'